src/main.rs

```
@@ -19,19 +38,87 @@ lazy_static! {
 // static client: Client = Client::new("url", Auth::UserPass("user".to_owned(), "password".to_owned())).unwrap();
 
 // TODO: Task 1
-fn time_to_mine(block_height: u64) -> Duration {
+fn time_to_mine(block_height: u64) -> Result<Duration, Error> {
     // * is a deref operator which invokes the Deref trait of the type RPC_CLIENT which was created
     // when the lazy macro is expanded
     // if a value has a static lifetime then it means that value lives as long as the program lives
     let rpc_client: &Client = &*RPC_CLIENT;
-    rpc_client.get_block_hash(234);
-    todo!()
+
+    let block_count = rpc_client.get_block_count()?;
+
+    if block_height > block_count {
```

Since you are returning a Result, it is contradictory to panic.
Rather return a corresponding error message